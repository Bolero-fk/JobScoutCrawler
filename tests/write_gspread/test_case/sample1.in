[{"company_name": "LMNエンタープライズ株式会社", "min_salary": 150, "max_salary": 400, "location": "愛知県", "using_lang": "PHP html css", "description": "テスト説明5テスト説明5テスト説明5テスト説明5", "recieve_date": "2023/05/13", "remaining_days": "2023/03/22", "site_name": "paiza"}, {"company_name": "ABC商事株式会社", "min_salary": 250, "max_salary": 600, "location": "大阪府", "using_lang": "Java", "description": "テスト説明2テスト説明2テスト説明2テスト説明2", "recieve_date": "2023/02/12", "remaining_days": "2023/04/12", "site_name": "paiza"}, {"company_name": "123サービス株式会社", "min_salary": 280, "max_salary": "- - -", "location": "北海道", "using_lang": "Ruby JavaScript", "description": "テスト説明3テスト説明3テスト説明3テスト説明3", "recieve_date": "2023/01/10", "remaining_days": "2023/04/14", "site_name": "paiza"}, {"company_name": "XYZ株式会社", "min_salary": 300, "max_salary": 800, "location": "東京都", "using_lang": "Python3", "description": "テスト説明1テスト説明1テスト説明1テスト説明1", "recieve_date": "2023/03/15", "remaining_days": "2023/04/20", "site_name": "paiza"}, {"company_name": "PQRコンサルティング株式会社", "min_salary": 200, "max_salary": 500, "location": "神奈川県", "using_lang": "C C++ C#", "description": "テスト説明4テスト説明4テスト説明4テスト説明4", "recieve_date": "2023/04/25", "remaining_days": "2023/04/03", "site_name": "paiza"}, {"company_name": "株式会社あいうえお", "min_salary": "- - -", "max_salary": "- - -", "location": "福岡県", "using_lang": "- - -", "description": "サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6", "recieve_date": "2023/03/09", "remaining_days": "2023/03/27", "site_name": "paiza"}, {"company_name": "株式会社CCC", "min_salary": "- - -", "max_salary": "- - -", "location": "東京都渋谷区", "using_lang": "- - -", "description": "タイトル5\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nバイオテクノロジーサイエンティスト", "recieve_date": "- - -", "remaining_days": "- - -", "site_name": "mynavi"}, {"company_name": "社名非公開（※社名は転職・就職相談にて開示）", "min_salary": 400, "max_salary": 1000, "location": "東京都豊島区／東京本社 石川県金沢市／ｏｏ営業所 神奈川県川崎市／ｏｏ営業所 茨城県つくば市／ｏｏ営業所 福岡県福岡市", "using_lang": "- - -", "description": "タイトル2\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nソフトウェアエンジニア,テストエンジニア,データサイエンティスト,フロントエンドエンジニア,プロジェクトマネージャー", "recieve_date": "- - -", "remaining_days": "- - -", "site_name": "mynavi"}, {"company_name": "株式会社AAA", "min_salary": 100, "max_salary": 234, "location": "東京都千代田区 または 神奈川県横浜市 ※勤務地の希望は考慮します。", "using_lang": "- - -", "description": "タイトル3\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\n生産技術", "recieve_date": "- - -", "remaining_days": "- - -", "site_name": "mynavi"}, {"company_name": "社名非公開（※社名は転職・就職相談にて開示）", "min_salary": 300, "max_salary": "- - -", "location": "東京都世田谷区／東京本社 補足情報1 補足情報2", "using_lang": "- - -", "description": "タイトル1\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】【ラベル10】\n●アピール1 ●アピール2 ●アピール3 ●アピール4 ●アピール5 ●アピール6\n研究開発・特許・品質管理・その他（ソフトウェア・ネットワーク）", "recieve_date": "- - -", "remaining_days": "- - -", "site_name": "mynavi"}, {"company_name": "株式会社BBB", "min_salary": 90, "max_salary": 678, "location": "大阪市北区 北海道札幌市 ※勤務地の希望は考慮します。", "using_lang": "- - -", "description": "タイトル4\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nセキュリティエンジニア", "recieve_date": "- - -", "remaining_days": "- - -", "site_name": "mynavi"}]