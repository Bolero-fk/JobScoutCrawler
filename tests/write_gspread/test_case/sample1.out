[["id", "会社名", "提示最低年収", "提示最高年収", "勤務地", "使用言語", "詳細", "受信日", "返答期日", "取得サイト"], ["1", "LMNエンタープライズ株式会社", "150", "400", "愛知県", "PHP html css", "テスト説明5テスト説明5テスト説明5テスト説明5", "2023/05/13", "2023/03/22", "paiza"], ["2", "ABC商事株式会社", "250", "600", "大阪府", "Java", "テスト説明2テスト説明2テスト説明2テスト説明2", "2023/02/12", "2023/04/12", "paiza"], ["3", "123サービス株式会社", "280", "- - -", "北海道", "Ruby JavaScript", "テスト説明3テスト説明3テスト説明3テスト説明3", "2023/01/10", "2023/04/14", "paiza"], ["4", "XYZ株式会社", "300", "800", "東京都", "Python3", "テスト説明1テスト説明1テスト説明1テスト説明1", "2023/03/15", "2023/04/20", "paiza"], ["5", "PQRコンサルティング株式会社", "200", "500", "神奈川県", "C C++ C#", "テスト説明4テスト説明4テスト説明4テスト説明4", "2023/04/25", "2023/04/03", "paiza"], ["6", "株式会社あいうえお", "- - -", "- - -", "福岡県", "- - -", "サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6", "2023/03/09", "2023/03/27", "paiza"], ["7", "株式会社CCC", "- - -", "- - -", "東京都渋谷区", "- - -", "タイトル5\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nバイオテクノロジーサイエンティスト", "- - -", "- - -", "mynavi"], ["8", "社名非公開（※社名は転職・就職相談にて開示）", "400", "1000", "東京都豊島区／東京本社 石川県金沢市／ｏｏ営業所 神奈川県川崎市／ｏｏ営業所 茨城県つくば市／ｏｏ営業所 福岡県福岡市", "- - -", "タイトル2\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nソフトウェアエンジニア,テストエンジニア,データサイエンティスト,フロントエンドエンジニア,プロジェクトマネージャー", "- - -", "- - -", "mynavi"], ["9", "株式会社AAA", "100", "234", "東京都千代田区 または 神奈川県横浜市 ※勤務地の希望は考慮します。", "- - -", "タイトル3\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\n生産技術", "- - -", "- - -", "mynavi"], ["10", "社名非公開（※社名は転職・就職相談にて開示）", "300", "- - -", "東京都世田谷区／東京本社 補足情報1 補足情報2", "- - -", "タイトル1\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】【ラベル10】\n●アピール1 ●アピール2 ●アピール3 ●アピール4 ●アピール5 ●アピール6\n研究開発・特許・品質管理・その他（ソフトウェア・ネットワーク）", "- - -", "- - -", "mynavi"], ["11", "株式会社BBB", "90", "678", "大阪市北区 北海道札幌市 ※勤務地の希望は考慮します。", "- - -", "タイトル4\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nセキュリティエンジニア", "- - -", "- - -", "mynavi"]]